modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/portlet/action/ExportTranslationMVCResourceCommand.java

```diff
@@ -164,6 +154,16 @@ private void _addZipEntry(
 					getTranslationInfoItemFieldValuesExporterOptional(
 						exportMimeType);
 
+		InfoItemFieldValuesProvider<Object> infoItemFieldValuesProvider =
+			_infoItemServiceTracker.getFirstInfoItemService(
+				InfoItemFieldValuesProvider.class, className);
+
+		InfoItemObjectProvider<Object> infoItemObjectProvider =
+			_infoItemServiceTracker.getFirstInfoItemService(
+				InfoItemObjectProvider.class, className);
+
+		Object object = infoItemObjectProvider.getInfoItem(classPK);
```

Comment:
As I say on https://github.com/liferay-lima/liferay-portal/pull/2532 the class created on that PR (InfoItemObjectProviderUtil) can be used to obtain the object and to reuse the code